gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/datafetcher/QueryServiceEntityFetcher.java

```diff
@@ -321,11 +326,6 @@ public EntityFetcherResponse getEntitiesAndAggregatedMetrics(
           ColumnMetadata metadata = chunk.getResultSetMetadata().getColumnMetadata(i);
           org.hypertrace.core.query.service.api.Value columnValue = row.getColumn(i);
           // add entity attributes from selections
-          addEntityAttribute(entityBuilder,
```

Comment:
Were there any additional unnecessary columns being added here?

Reply:
This was a bug. Trying to turn an aggregated metric into an attribute. Caused this warn log observed in gw service:

```
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for AVGRATE_SERVICE.numCalls_[1]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for PERCENTILE_SERVICE.duration_[99]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for SUM_SERVICE.errorCount_[]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for AVG_SERVICE.errorCount_[]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for PERCENTILE_SERVICE.duration_[50]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for SUM_SERVICE.numCalls_[]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for AVG_SERVICE.numCalls_[]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for AVGRATE_SERVICE.numCalls_[1]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for PERCENTILE_SERVICE.duration_[99]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for SUM_SERVICE.errorCount_[]
2020-08-11 11:32:48.898 [grpc-default-executor-401] WARN  o.h.g.s.c.c.QueryAndGatewayDtoConverter - No attribute metadata found for AVG_SERVICE.errorCount_[]
```